src/test/java/com/metamx/emitter/core/HttpEmitterConfigTest.java

```diff
@@ -40,6 +40,7 @@ public void testDefaults()
     Assert.assertEquals(BatchingStrategy.ARRAY, config.getBatchingStrategy());
     Assert.assertEquals(5 * 1024 * 1024, config.getMaxBatchSize());
     Assert.assertEquals(250 * 1024 * 1024, config.getMaxBufferSize());
+    Assert.assertEquals(Long.MAX_VALUE, config.getFlushTimeOut());
```

Comment:
I'd rather keep with current test convention unless there's a compelling reason to change it.